CoJezdzi/AppFramework/_Networking/Request/UMWarsawTransportApi.swift

```diff
@@ -0,0 +1,32 @@
+import APIKit
+
+final class UMWarsawTransportApi {
+    
+    private init() {
+    }
+    
+    struct GetTransportMeanRequest: UMWarsawRequest {
+        
+        let vehicleType: WarsawVehicleType
+
+        // MARK: RequestType
+        typealias Response = UMWarsawTransportResponse
+        
+        var method: HTTPMethod {
+            return .post
+        }
+        
+        var path: String {
```

Comment:
This one I like very much! I have used whole this part as "base" because all API calls go thru this url ;)